packages/rs-drive/src/drive/grove_operations/grove_clear/v0/mod.rs

```diff
@@ -16,10 +17,35 @@ impl Drive {
             allow_deleting_subtrees: false,
             trying_to_clear_with_subtrees_returns_error: false,
         };
+
+        let maybe_path_for_logs = if tracing::event_enabled!(Level::TRACE) {
+            Some(path.clone())
+        } else {
+            None
+        };
+
         // we will always return true if there is no error when we don't check for subtrees
-        self.grove
+        let result = self
+            .grove
             .clear_subtree(path, Some(options), transaction)
             .map_err(Error::GroveDB)
-            .map(|_| ())
+            .map(|_| ());
+
+        if tracing::event_enabled!(Level::TRACE) && result.is_ok() {
+            let root_hash = self
+                .grove
+                .root_hash(transaction)
+                .unwrap()
+                .map_err(Error::GroveDB)?;
+
+            tracing::trace!(
+                path = ?maybe_path_for_logs.unwrap().to_vec(),
```

Comment:
maybe to_hex instead of to_vec?

Reply:
We log the GroveDB batch, which has a deeply nested structure and it uses vectors I believe. There is no easy way to change this structure to make hexes as well. I would prefer to have paths everywhere in the same format. I would probably make a separate PR where we revisit all log fields in order to make them more human readable.